tests/sentry/event_manager/interfaces/snapshots/test_exception/test_slim_exception_data_over_max.pysnap

```diff
@@ -1,49 +1,44 @@
 ---
-created: '2019-03-14T17:12:35.455057Z'
+created: '2019-05-22T14:45:11.531485Z'
 creator: sentry
 source: tests/sentry/event_manager/interfaces/test_exception.py
 ---
 values:
 - stacktrace:
     frames:
-    - abs_path: exc 0 frame 0
-      context_line: b
+    - context_line: b
```

Comment:
Where are the abspaths?

Reply:
https://github.com/getsentry/semaphore/pull/203

Reply:
This particular diff is because normalization does not run at all for this test.